### ISSUE TITLE
podalize only processes via CPU

### ISSUE THREAD

**Issue by naturian** [mave5/podalize#2]
Hey there,

i finally made podalize running with Windows 10. The only issue that remains, that every single process of podalize is happening over the CPU. CUDA toolkit and pytorch cuda is installed. Does podalize automatically detects if CUDA is available?

Best,
Marcel

**Comment by mave5** [comment]
@naturian Nice making it work  on Windows. You can share the how as a pull request.
On Ubuntu, it automatically detects my GPU. 
Try to see if PyTorch can detect your GPU in a separate script:

```
>>> import torch
>>> torch.cuda.is_available()
True
>>> torch.cuda.current_device()
0
>>> torch.cuda.get_device_name()
'NVIDIA GeForce GTX TITAN X'

```

**Comment by naturian** [comment]
I think i've installed the cpu only repos. got to figure it out. i'll do a pull request when i fixed the problem. nice script btw!

**Comment by hamisus** [comment]
Any idea how to get it running on mps with mac m1? Was very impressed with the quality and accuracy of the output, but it took about 12 hours to get it to output one transcript

**Comment by oluckyman** [comment]
Hey, @donzthefonz! any progress with M1?
